src/ui/components/results/resultsheadercomponent.js

```diff
@@ -113,10 +113,18 @@ export default class ResultsHeaderComponent extends Component {
     }));
   }
 
+  /**
+   * Returns an array of object the handlebars can understand and render
+   * the applied filters bar from.
+   * @returns {Array<Object>}
+   */
+  _getAppliedFiltersArray () {
+    return this._convertGroupedFiltersToArray(this._groupAppliedFilters());
```

Comment:
Is this the only place convertGroupdFiltersToArray is called? If so, I think we should just consolidate that method into this one. This method's not doing too much currently.

Reply:
sure that's simpler